src/docfx/build/xref/XrefResolver.cs

```diff
@@ -232,7 +232,7 @@ private static string RemoveSharingHost(string url, string hostName)
             if (_internalXrefMap.Value.TryGetValue(uid, out var specs))
             {
                 var spec = default(InternalXrefSpec);
-                if (!monikers.HasValue || !monikers.Value.HasMonikers)
+                if (specs.Length == 1 || !monikers.HasValue || !monikers.Value.HasMonikers)
```

Comment:
What exactly is the rule here? If we allow fallback to other versions (or better yet latest version), should line 244 also fallback?